### DISCUSSION TITLE
Manually preload images

### DISCUSSION THREAD

**Discussion by tablufc88** [vercel/next.js#27808] [Help]
Is there a way of manually preloading images in Next? I'm using a different logo depending on light and dark mode. When I set priority="true" on the image, it works but preloads the wrong one.

Here is my logo component. If anyone could help it would be appreciated 

```javascript
const Logo = () => {
  const { theme } = React.useContext(ThemeContext)
  let src
  theme === "dark" ? (src = GD.brand.altLogo) : (src = GD.brand.logo)
  return (
    <div tw="width[calc(150px + 5vw)] my-3 flex">
      <ImageWrapper hash={src.blurHash} ext={src.ext}>
        <Image
          src={src.url}
          alt={src.alternativeText || "Logo"}
          height={src.height}
          width={src.width}
          priority="true"
        />
      </ImageWrapper>
    </div>
  )
}
```

**Comment by diegoaguilar** [comment]
Hi @tablufc88 did you get a solution or final decision on this? While not the same issue, I'm about to launch a prod site and having some .... concerns on the perks and offs for next/image

**Reply by tablufc88** [reply]
Hi @diegoaguilar, I love the next/image, It makes life so much easier and is great for SEO. I was just having issues with preloading. I would normally manually preload any above the fold images, but I was wondering whether there is a way to do it without priority="true"
